src/main/java/org/folio/circulation/resources/DeclareClaimedReturnedItemAsMissingResource.java

```diff
@@ -23,6 +32,9 @@
 
 public class DeclareClaimedReturnedItemAsMissingResource extends Resource {
 
+  private static final String NOTE_MESSAGE = "Claimed returned item marked missing";
+  private static final String NOTE_DOMAIN  = "loans";
```

Comment:
If these are only used in the `createNote` method, could they be moved to be part of the scope of that method?